mne/decoding/transformer.py

```diff
@@ -168,35 +170,34 @@ def __init__(self, info=None):
         self.n_channels = None
         self.n_times = None
 
-    def fit(self, epochs_data, y):
+    def fit(self, epochs, y=None):
```

Comment:
There is a deprecation. Perhaps we can make it to accept both `epochs` as well as `epochs._data`.